source/mbed_trace.c

```diff
@@ -304,7 +304,7 @@ static int8_t mbed_trace_skip(int8_t dlevel, const char *grp)
 }
 static void mbed_trace_default_print(const char *str)
 {
-    puts(str);
+    printf(str);
```

Comment:
tsss... how could I write this line? Obviously, calling something like `tr_info("%%s")` is going to crash.